src/integrationTest/resources/tests/audio/MediaControllerPostAdminMediasSearchIntTest/multipleAndHearingStartAndEnd.json

```diff
@@ -0,0 +1,38 @@
+[
+  {
```

Comment:
Would expect the `id` to be part of the response too.

Reply:
it is, but we cannot assert on it during a integration test as it will change every time its run, so it is removed.

Reply:
i have now added id checks